src/plasma/plasma_manager.c

```diff
@@ -563,23 +569,17 @@ void send_queued_request(event_loop *loop,
   }
 
   plasma_request_buffer *buf = conn->transfer_queue;
-  plasma_request manager_req = plasma_make_request(buf->object_id);
   switch (buf->type) {
   case PLASMA_TRANSFER:
-    memcpy(manager_req.addr, conn->manager_state->addr,
-           sizeof(manager_req.addr));
-    manager_req.port = conn->manager_state->port;
-    CHECK(plasma_send_request(conn->fd, PLASMA_TRANSFER, &manager_req) >= 0);
+    CHECK(plasma_send_DataRequest(conn->fd, state->builder, buf->object_id, state->addr, state->port) >= 0);
```

Comment:
Why is this a DataRequest, shouldn't it be a TransferRequest? Or did you change the naming convention here?